packages/phone-number-privacy/combiner/src/database/wrappers/account.ts

```diff
@@ -68,18 +49,14 @@ export async function setDidMatchmaking(account: string) {
       return insertRecord(newAccount)
     }
   } catch (e) {
-    logger.error(ErrorMessages.DATABASE_UPDATE_FAILURE, e)
-    return true
+    logger.error('Error setting matchmaking status', e)
+    throw new Error(ErrorMessage.DATABASE_UPDATE_FAILURE)
   }
 }
 
 async function insertRecord(data: Account) {
-  try {
-    await accounts()
-      .insert(data)
-      .timeout(10000)
-  } catch (e) {
-    logger.error(ErrorMessages.DATABASE_INSERT_FAILURE, e)
-  }
+  await accounts()
+    .insert(data)
+    .timeout(10000)
```

Comment:
I know this isn't new in this PR, but can we define this `10000` as a constant?

Reply:
Can do